src/ansi-c/cprover_builtin_headers.h

```diff
@@ -10,6 +10,7 @@ __CPROVER_bool __CPROVER_equal();
 __CPROVER_bool __CPROVER_same_object(const void *, const void *);
 __CPROVER_bool __CPROVER_is_invalid_pointer(const void *);
 _Bool __CPROVER_is_zero_string(const void *);
+__CPROVER_bool __CPROVER_is_list();
```

Comment:
We need to find some way of documenting this (though it currently would not actually have any effect).

Reply:
The only use-case right now is contract checking, and the documentation should be nearby.

Reply:
Perhaps a single line of commentary could be provided next to this declaration?

Reply:
ok, added